chalicelib/api/events_rush.py

```diff
@@ -49,9 +49,9 @@ def checkin_rush(event_id):
     return events_rush_service.checkin_rush(event_id, data)
 
 
-@events_rush_api.route("/events/rush/default", methods=["GET"], cors=True)
-def get_rush_events_default_category():
-    return events_rush_service.get_rush_events_default_category()
+@events_rush_api.route("/events/rush/default/{email}", methods=["GET"], cors=True)
```

Comment:
So usually putting sensitive details on the route itself is considered not the best practice. It would be better to embed that email in a body and use that to send it to the backend – could you refactor this?

Reply:
Yes for sure!

Reply:
That is good to know :)